CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_tiktok_ads v0.7.0
+[PR #22](https://github.com/fivetran/dbt_tiktok_ads/pull/22) includes the following updates:
+
+## Breaking Changes
```

Comment:
We should callout that these are upstream breaking changes. Since the `age_groups` column isn't used in any models within dbt_tiktok_ads it could be confusing to read this and wonder where the age_groups column was before. We should clearly callout that this change really only impacts the upstream staging model.

Reply:
Updated!